bears/templates/meta.yaml.in

```diff
@@ -0,0 +1,16 @@
+package:
+  name: $NAME
+  version: $VERSION
+
+source:
+  git_url: $URL
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
Isn't setuptools also a run depenedency as we use pkg_resource for entrypoints ( which is a part of setuptools )